### DISCUSSION TITLE
Questions on porous media Navier-Stokes governing equations

### DISCUSSION THREAD

**Discussion by JuliaHLee** [idaholab/moose#23918] [Q&A Modules: Navier-Stokes] [answered]
Dear MOOSE developers,

I have been working on heat transport MOOSE model with porous media Navier-Stokes (PINSFV) equations. I want the model to numerically solve two-phase energy equations from Wakao and Kaguei (1982). I thought the PINSFV equations on the website very similar to the one of the literature. So I started to use PINSFV. However, while I was comparing in detail, I found some differences between them. Here I would like to ask for help to understand the equations used in MOOSE in detail. 

The PINSFV energy equations, which I captured on the webpage, are:
![image](https://user-images.githubusercontent.com/12911783/229133686-4ecf1cac-6b04-4e52-89bf-4c9c00ff6555.png)

Two energy equations from Wakao and Kaguei (1982), rewritten with MOOSE symbols, are:
![image](https://user-images.githubusercontent.com/12911783/229138300-ce274ae8-c9e5-40f9-b728-0cf05108d463.png)

The differences I would like to discuss are below:
1)	In MOOSE fluid phase energy equation, I understood that (v_D/epsilon) corresponds to v in the literature. Then, what is a physical meaning behind the additional rho in numerator in the advection term?
2)	If I consider porosity to be constant, porosity in convective heat transfer term will be cancelled out. Then this term misses the factor, epsilon or (1-epsilon), in the denominator, compared to the one of literature. Would there be a reason?
3)	Convective heat transfer term in MOOSE equation misses the factor, specific surface area a_sf. So I assumed alpha equals to h_sf multiplied by a_sf. Is there any correlation programmed in MOOSE for such a parameter?

I plan to add functions in my input file, in case a parameter needs correlation. So I would like to have more information on the PINSFV equations. Can I have a reference of the PINSFV governing equation?

Many thanks in advance!

Best regards,

Julia

**Comment by lindsayad** [comment, marked as answer]
> 1. In MOOSE fluid phase energy equation, I understood that (v_D/epsilon) corresponds to v in the literature. Then, what is a physical meaning behind the additional rho in numerator in the advection term?

That is a bug in the documentation. That prefactor of rho / epsilon should not be there. The prefactor does not exist in the code

> 2\. If I consider porosity to be constant, porosity in convective heat transfer term will be cancelled out. Then this term misses the factor, epsilon or (1-epsilon), in the denominator, compared to the one of literature. Would there be a reason?

This is another documentation bug. The epsilon prefactor for the fluid equation and the (1 - epsilon) prefactor for the solid equation do not exist in the code

> 3\. Convective heat transfer term in MOOSE equation misses the factor, specific surface area a_sf. So I assumed alpha equals to h_sf multiplied by a_sf. Is there any correlation programmed in MOOSE for such a parameter?

I will have to ask our Pronghorn users about this one. @GiudGiud @snschune @pbalest do you guys have any comment on this item?

@JuliaHLee what documentation page were you looking at?

**Reply by JuliaHLee** [reply]
Thank you very much for the reply!

I found the PINSFV equations here: https://mooseframework.inl.gov/modules/navier_stokes/pinsfv.html

Is there any specific reference of the equations used in the code?

**Reply by GiudGiud** [reply]
The equations in the documentation should have matched the code but it seems they don't. 
For 3., we have not merged any correlations in the Navier Stokes module, they are all in Pronghorn.

**Reply by lindsayad** [reply]
See https://github.com/idaholab/moose/pull/23934 for updates to the documentation

**Reply by JuliaHLee** [reply]
Thanks a lot for the answer and updates!
